opentracing-example/src/main/scala/zio/telemetry/opentracing/example/http/Client.scala

```diff
@@ -21,14 +19,13 @@ case class Client(backend: Backend, config: AppConfig) {
   ): Task[Statuses] =
     for {
       url      <- ZIO.fromEither(backendUrl)
-      response <- backend
-                    .send(
-                      basicRequest
-                        .get(url.withPath("status"))
-                        .headers(headers)
-                        .response(asJson[Status])
-                    )
-      status    = response.body.getOrElse(Status.down("backend"))
+      request   = Request
+                    .get(url)
```

Comment:
`url` is missing the path:
`.get(url.withPath("status"))`

Reply:
fixed